btc-lsp/src/BtcLsp/Psbt/PsbtOpener.hs

```diff
@@ -117,7 +129,9 @@ openChannelPsbt utxos toPubKey changeAddress lspFee = do
       void . T.atomically . T.writeTChan chan $ LndSubFail
   case res of
     Left e -> throwE $ FailureInternal $ inspect e
-    Right _ -> fundStep pcid chan
+    Right _subA -> do
```

Comment:
Write something like 'Right {}` on pattern matching if value is unused, `_` prefix suppresses warnings, which is not the best behaviour.